classes/autotools.oeclass

```diff
@@ -48,7 +48,22 @@ autotools_configure[emit] = "do_configure"
 do_configure () {
     do_configure_autotools
 }
+
+AUTOCONFDIR = ""
+S_AUTOCONFDIR = ""
```

Comment:
You probably need to add a comment on the meaning of these two variables, so there is a chance that people will figure out how to use this.